### PR TITLE
Feature/improve mgmnt group id desc

### PR DESCRIPTION
**Pull request by tfelix**

### CODE REVIEW

**Review by felixzieger — APPROVED** [review]

File: docs/meshstack.azure.config.md [hunk 1]
```diff
@@ -234,6 +234,8 @@ meshStack allows operators to control how the replicator provisions subscription
 
 Pre-provisioned subscriptions are useful when no EA is available and operators want to consume CSP or Pay-as-you-go Subscriptions instead.
 
+> **Important**: It is not advisable to re-use Subscriptions for multiple meshProjects (for example when you marked a meshProject for deletion but did not properly delete the Subscription in Azure but and instead re-use them for pre-provisioning). If you do this, the metering system will assign the Subscription to the wrong meshProject for billing calculation. Double billed resources will be the consequences.
```
Comment (chewbeccy) [inline docs/meshstack.azure.config.md:237]:
```suggestion
> **Important**: It is not supported to re-use Subscriptions for multiple meshProjects (for example when you marked a meshProject for deletion but did not properly delete the Subscription in Azure but and instead re-use them for pre-provisioning). If you do this, the metering system will assign the Subscription to the wrong meshProject for billing calculation. Double billed resources will be the consequence.
```

File: docs/meshstack.azure.config.md [hunk 2]
```diff
@@ -265,13 +267,17 @@ let EnterpriseEnrollment =
                 The Microsoft Subscription offer type to use when creating subscriptions.
```
Comment (chewbeccy) [inline docs/meshstack.azure.config.md:267]: sometimes Subscription is written in upper case sometimes lower case

File: docs/meshstack.azure.landing-zones.md [hunk 3]
```diff
@@ -3,18 +3,15 @@ id: meshstack.azure.landing-zones
 title: Landing Zones
 ---
 
-In Azure, a [Landing Zone](./meshcloud.landing-zones.md) is defined via a management group the subscription for the project will be assigned to. Policies can be applied
-to these management groups. Optionally a blueprint can also be defined. Via an Azure Blueprint default resources can be deployed to the
-subscription and additional specific policies can be defined. A blueprint can be configured to decline users to change or delete the
-resources and policies created by the blueprint.
+In Azure, a [Landing Zone](./meshcloud.landing-zones.md) is defined via a management group the subscription for the project will be assigned to. Policies can be applied to these management groups. Optionally a blueprint can also be defined. Via an Azure Blueprint default resources can be deployed to the subscription and additional specific policies can be defined. A blueprint can be configured to decline users to change or delete the resources and policies created by the blueprint.
```
Comment (chewbeccy) [inline docs/meshstack.azure.landing-zones.md:6]: upper and lower case mixed for management group and blueprint.
Lets make it upper case everywhere

**Review by chewbeccy — CHANGES_REQUESTED** [review]
upper and lower case spelling and a small suggestion in one file

File: docs/meshstack.azure.config.md (hunk 1 above)
Comment (tfelix) [reply docs/meshstack.azure.config.md:237]: adapted.

File: docs/meshstack.azure.landing-zones.md (hunk 3 above)
Comment (tfelix) [reply docs/meshstack.azure.landing-zones.md]: Some words are coming from the Dhall config. I fix these in the PR for the manegemtn group desc.

**Review by chewbeccy — APPROVED** [review]

**Pull request merged**